src/coprocessor/endpoint.rs

```diff
@@ -299,7 +301,11 @@ impl Host {
                         future::ok::<_, futures_mpsc::SendError<_>>((resp, (ctx, finished)))
                     })
                 });
-                on_resp.respond_stream(box s, pool.clone());
+
+                // TODO: Add test to ensure back pressure works.
+                let (tx, rx) = futures_mpsc::channel(self.stream_channel_size);
```

Comment:
It seems the stream response transferred with one more channel( futures_mpsc::channel+paired_future_callback) than before(futures_mpsc::channel)?